### ISSUE TITLE
Add learning rate scheduler

### ISSUE THREAD

**Issue by hobogalaxy** [kinoai/deep-learning-project-template#21]

**Comment by hobogalaxy** [comment]
https://pytorch-lightning.readthedocs.io/en/stable/optimizers.html
